cfpb/consumerfinance.gov
# ROUT: Replace hardcoded icons with actual DS source icons

## Changes

- ROUT: Replace hardcoded icons with actual DS source icons


## How to test this PR

1. Visit http://localhost:8000/rural-or-underserved-tool/ and enter an address and click "Check addresses"
2. The result show/hide map toggle should have icons.


## Screenshots

<img width="126" alt="Screen Shot 2023-03-17 at 1 36 56 PM" src="https://user-images.githubusercontent.com/704760/225978090-b2416b76-e45b-4915-9947-7b15b423ffca.png">


<img width="124" alt="Screen Shot 2023-03-17 at 1 33 36 PM" src="https://user-images.githubusercontent.com/704760/225978014-7660fc14-a175-4920-b759-93aa9935813d.png">
